code/modules/shuttle/computers/dropship_computer.dm

```diff
@@ -299,7 +298,14 @@
 		hivenumber = xeno.hivenumber
 	xeno_message(SPAN_XENOANNOUNCE("The Queen has commanded the metal bird to depart for the metal hive in the sky! Rejoice!"), 3, hivenumber)
 	xeno_message(SPAN_XENOANNOUNCE("The hive swells with power! You will now steadily gain pooled larva over time."), 2, hivenumber)
-	GLOB.hive_datum[hivenumber].abandon_on_hijack()
+	var/datum/hive_status/hive = GLOB.hive_datum[hivenumber]
+	hive.abandon_on_hijack()
+	var/original_evilution = hive.evolution_bonus
+	hive.override_evilution(10, TRUE)
+	if(hive.living_xeno_queen)
+		var/datum/action/xeno_action/onclick/grow_ovipositor/ovi_ability = get_xeno_action_by_type(hive.living_xeno_queen, /datum/action/xeno_action/onclick/grow_ovipositor)
+		ovi_ability.reduce_cooldown(5 MINUTES)
```

Comment:
It would be better to reduce by the cooldown amount for better future proofing. E.g. maybe ovi cooldown nerfed to 10 minutes, then this will only reduce the time an amount not reset it.
```suggestion
		ovi_ability.reduce_cooldown(ovi_ability.xeno_cooldown)
```
Untested suggestion, but should be correct.